key-manager-oapi.yaml

```diff
@@ -0,0 +1,199 @@
+openapi: "3.0.3"
+
+info:
+  title: "Eth2 key manager API"
+  description: |
+    API specification for a key manager client, which enables users to manage keystores.
+
+    The key manager API is served by the binary holding the validator keys. This binary may be a remote signer or a validator client.
+
+    All routes SHOULD be exposed through a secure channel, such as with HTTPs, an SSH tunnel, a VPN, etc.
+
+    All requests by default send and receive JSON, and as such should have either or both of the "Content-Type: application/json"
+    and "Accept: application/json" headers.
+
+    All sensitive routes are be authenticated with a token. This token should be provided by the user via a secure channel:
+      - Log the token to stdout when running the binary with the key manager API enabled
+      - Read the token from a file available to the binary
+  version: "Dev - Eth2Spec v1.0.1"
+  contact:
+    name: Ethereum Github
+    url: https://github.com/ethereum/eth2.0-apis/issues
+  license:
+    name: "Apache 2.0"
+    url: "https://www.apache.org/licenses/LICENSE-2.0.html"
+
+servers:
+  - url: "{server_url}"
+    variables:
+      server_url:
+        description: "key manager API url"
+        default: "http://public-mainnet-node.ethereum.org/api"
+
+tags:
+  - name: Accounts
+    description: Endpoints to manage key manager accounts.
+  - name: SlashingProtection
+    description: Endpoints to import and export slashing protection db data.
+  - name: AccountAuth
+    description: Endpoints to authenticate accounts requests.
+
+paths:
+  /eth/v1/accounts:
```

Comment:
should this api be called accounts? I was wondering if keys or some other key around keys makes more sense to me... at least for internal discussion accounts gets confused with the webui account/user. What do others think? It was one of the terms we wanted to review on the Prysm side at least.

Reply:
/eth/v1/keys sounds good to me at least.